resources/views/processes/scripts/edit.blade.php

```diff
@@ -70,6 +100,7 @@
     <script>
       new Vue({
         el: '#editScript',
+        mixins: addons,
```

Comment:
Same here.

Reply:
yes, it's arming with a controller and passed that way.